src/js/components/artifacts/releaseslist.js

```diff
@@ -13,17 +13,36 @@ const filters = ['device_types_compatible', 'descriptions', 'Name'];
 
 const filter = createFilterOptions({ stringify: option => filters.map(item => option[item]).join(' ') });
 
+const sortingOptions = [
+  { title: 'Name', value: 'Name' },
+  { title: 'Modification Date', value: 'latestModified' }
```

Comment:
I'd suggest "Date modified" for the label, it's a bit more common phrasing. Nice addition though!

Reply:
Should now be "Date modified" 👌